std/manual.md

```diff
@@ -771,7 +771,7 @@ source /usr/local/etc/bash_completion.d/deno.bash
 
 V8 has many many internal command-line flags, that you can see with
 `--v8-options`.
-[It looks like this.](https://gist.github.com/ry/1c5b080dcbdc6367e5612392049c9ee7)
+[It looks like this.](https://gist.github.com/josephsintum/f7877cd6697a0d3158985fc382c98707)
```

Comment:
This isn't ideal, nor is linking to `lib.deno_runtime.d.ts` on master for `deno types`.

Maybe these kinds of immutable outputs could be deployed to the site on release or something, but honestly are these links even helpful?

Reply:
Use https://gist.github.com/ry/a610ce48cba2f0225f9c81a5a833fc87

Reply:
Got it 
adding it